packages/wallet/backend/src/walletAddressKeys/service.ts

```diff
@@ -91,7 +92,15 @@ export class WalletAddressKeyService implements IWalletAddressKeyService {
       walletAddressId
     }
 
-    await WalletAddressKeys.query().insert(key)
+    try {
+      await WalletAddressKeys.query().insert(key)
```

Comment:
@raducristianpopa For the sake of having a unique ID for each key, every user can have one unique key for uploading, ensuring that others can't upload the same key.